From a20f33e018e1d70537caf434d6c0ec0af304763f Mon Sep 17 00:00:00 2001
From: Daniel Nachun <daniel.nachun@gmail.com>
Date: Wed, 26 Nov 2025 02:04:35 -0800
Subject: [PATCH] minimal win makevars

---
 src/Makevars.win | 28 ++++++----------------------
 1 file changed, 6 insertions(+), 22 deletions(-)

diff --git a/src/Makevars.win b/src/Makevars.win
index 6c86060..0f86c78 100644
--- a/src/Makevars.win
+++ b/src/Makevars.win
@@ -1,26 +1,10 @@
-RWINLIB = ../.deps/libgit2
-TARGET = lib$(subst gcc,,$(COMPILED_BY))$(R_ARCH)
-
-ifneq ($(R_COMPILED_BY),gcc 8.3.0)
-PCRE=-lpcre2-8
-endif
-
-PKG_CPPFLAGS = -I$(RWINLIB)/include \
+PKG_CPPFLAGS =
+    -Lwin64 \
+    -I$(LIBRARY_PREFIX_CYG)/include \
 	-DR_NO_REMAP -DSTRICT_R_HEADERS
 
 PKG_LIBS = \
-	-L$(RWINLIB)/$(TARGET) \
-	-L$(RWINLIB)/lib \
-	-lgit2 -lssh2 -lssl -lcrypto $(PCRE) -lz -liconv \
+    -Lwin64 \
+    -L$(LIBRARY_PREFIX_CYG)/lib \
+	-lgit2 -lz -liconv \
 	-lwinhttp -lws2_32 -lcrypt32 -lole32 -lrpcrt4 -lsecur32
-
-all: $(SHLIB) cleanup
-
-# Needed for parallel make
-$(OBJECTS): | $(RWINLIB)
-
-$(RWINLIB):
-	@"${R_HOME}/bin${R_ARCH_BIN}/Rscript.exe" "../tools/winlibs.R"
-
-cleanup: $(SHLIB)
-	@rm -Rf $(RWINLIB)
-- 
2.51.0

